server/src/UseCases/Games/GetAllGames/GetAllGamesQueryHandler.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using BGS.ApplicationCore.Entities;
+using BGS.SharedKernel;
+using MediatR;
+
+namespace BGS.UseCases.Games.GetAllGames;
+
+public class GetAllGamesQueryHandler : IRequestHandler<GetAllGamesQuery, IEnumerable<Game>>
+{
+    private readonly IRepository<Game> _gameRepository;
+
+    public GetAllGamesQueryHandler(IRepository<Game> gameRepository)
+    {
+        _gameRepository = gameRepository;
+    }
+
+    public async Task<IEnumerable<Game>> Handle(GetAllGamesQuery request, CancellationToken cancellationToken)
+    {
+        return await _gameRepository.ListAsync(cancellationToken);
```

Comment:
https://cezarypiatek.github.io/post/async-analyzers-p1/#1-redundant-asyncawait also for `GetGameByIdQueryHandler`